middleware/fastcgi/fastcgi.go

```diff
@@ -128,15 +128,25 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error)
 	return h.Next.ServeHTTP(w, r)
 }
 
-func getClient(r *Rule) (*FCGIClient, error) {
-	// check if unix socket or TCP
+// Return the network and address of a given Rule
+func (r Rule) parseAddress() (string, string) {
```

Comment:
Even though the function is not exported, godoc-style comments should be used just for consistency. i.e. "parseAddress returns the network and address of r."